browser/ai_chat/upload_file_helper.cc

```diff
@@ -34,7 +35,36 @@ data_decoder::DataDecoder* GetDataDecoder() {
 namespace ai_chat {
 
 namespace {
-using UploadImageCallback = mojom::AIChatUIHandler::UploadImageCallback;
+using UploadFileCallback = mojom::AIChatUIHandler::UploadFileCallback;
+
+bool LooksLikePdf(base::span<const uint8_t> maybe_pdf_data) {
+  constexpr auto kPdfStartsWith = base::span_from_cstring("%PDF-");
+  return maybe_pdf_data.size() >= 50u &&
```

Comment:
nit: maybe add a comment for why there's 50 here.

Reply:
it is actually borrowed from printing_utils.h so I included it directly in db052348920037997bd564c04bf5d4d6505199bf